webapp/src/widgets/icons/edit.tsx

```diff
@@ -3,20 +3,13 @@
 
 import React from 'react'
 
-import './edit.scss'
+import CompassIcon from './compassIcon'
 
-export default function EditIcon(): JSX.Element {
+export default function DeleteIcon(): JSX.Element {
```

Comment:
I don't think this rename is correct. Should be `EditIcon` ?

Reply:
:facepalm: I don't know how that happened.